plugins/modules/mount.py

```diff
@@ -251,46 +251,35 @@ def is_fspath_mounted(module, mount_dir, mount_over_dir):
 
     if mount_over_dir:
         # when mounting NFS, make sure we check for the NFS
-        # client mount point
-        cmd = "lsfs -l %s" % mount_over_dir
+        fs_name = mount_over_dir
     elif mount_dir:
-        cmd = "lsfs -l %s" % mount_dir
```

Comment:
Should we give the full path of the command here for a safer side.

Reply:
i am no longer using lsfs on my new changes.
i am now relying only in df command.
i can make changes to have the full path of df cmd instead.

thanks